overcooked_ai_py/agents/agent.py

```diff
@@ -111,32 +116,32 @@ def __init__(self, agent):
     def joint_action(self, state):
         # Reduce computation by half if both agents are coupled planning agents
         joint_action_plan = self.a0.mlp.get_low_level_action_plan(state, self.a0.heuristic, delivery_horizon=self.a0.delivery_horizon, goal_info=True)
-        
+
         if len(joint_action_plan) == 0:
             return ((Action.STAY, {}), (Action.STAY, {}))
-        
+
         joint_action_and_infos = [(a, {}) for a in joint_action_plan[0]]
         return joint_action_and_infos
-            
+
 
 
 class AgentFromPolicy(Agent):
     """
     Defines an agent from a `state_policy` and `direct_policy` functions
     """
-    
-    def __init__(self, state_policy, direct_policy, stochastic=True, action_probs=False):
+
+    def __init__(self, state_policy, direct_policy, stochastic=True, return_action_probs=True):
         """
         state_policy (fn): a function that takes in an OvercookedState instance and returns corresponding actions
         direct_policy (fn): a function that takes in a preprocessed OvercookedState instances and returns actions
         stochastic (Bool): Whether the agent should sample from policy or take argmax
-        action_probs (Bool): Whether agent should return action probabilities or a sampled action
+        return_action_probs (Bool): Whether agent should return action probabilities or a sampled action
         """
         self.state_policy = state_policy
         self.direct_policy = direct_policy
         self.history = []
         self.stochastic = stochastic
-        self.action_probs = action_probs
+        self.return_action_probs = return_action_probs
```

Comment:
I should have done this when cleaning but missed it: remove action_probs entirely from this class, and always pass in action_probs=True when calling self.state_policy.